clang/test/dpct/cmake_migration/case_051/ref.txt

```diff
@@ -1 +1 @@
-// CHECK: Warning: CMake build script file like CMakeLists.txt is not found, so no CMake build script file will be migrated.
+// CHECK: Warning: CMake or Python setup build script file like CMakeLists.txt/package.cmake or setup.py is not found, so no CMake/ Python setup build script file will be migrated.
```

Comment:
Pls use a separate folder like py_setup_migration for python_setup migration feature, just like cmake_migration folder for cmake.